frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -676,11 +676,15 @@ export const PublicFormProvider = ({
       submitEmailModeFormFetchMutation,
       submitEmailModeFormMutation,
       enableEncryptionBoundaryShift,
+      enableVirusScanner,
+      submitStorageModeClearFormMutation,
```

Comment:
Are these vals being used in the content of the provider?

Reply:
Yup! enableVirusScanner used in [L560](https://github.com/opengovsg/FormSG/blob/e9bd3fcfaf8998e1a841373f4c5c7a92b217b99e/frontend/src/features/public-form/PublicFormProvider.tsx#L560), submitStorageModeClearFormMutation in [L608](https://github.com/opengovsg/FormSG/blob/e9bd3fcfaf8998e1a841373f4c5c7a92b217b99e/frontend/src/features/public-form/PublicFormProvider.tsx#L608)